front/public/locales/fr/stdcm.json

```
@@ -145,6 +145,43 @@
         "withOutputs": "Simulation n°{{id}}",
         "withoutOutputs": "Pas de capacité"
       },
+      "modal": {
+        "additionalInformation": "INFORMATIONS COMPLÉMENTAIRES",
+        "addComment": "Commentaire",
+        "arrivalPoint": "Arrivée à {{arrivalAt}}",
+        "arrivalTime": "Arrivée de la simulation retenue",
+        "arrivalTiming": "CONTRAINTE HORAIRE DE L'ARIVEE",
+        "Autre": "Autres parcours",
+        "cancel": "Annuler",
+        "close": "Fermer",
+        "comment": "Commentaire",
+        "compositionCode": "code de composition",
+        "convoy": "CONVOI",
+        "createGesicoRequest": "Créer la demande GESICO",
+        "csCode": "Code CS",
+        "departurePoint": "Départ {{departureAt}}",
+        "departureTime": "Départ de la simulation retenue",
+        "departureTiming": "CONTRAINTE HORAIRE DE DÉPART",
+        "followUpMessage": "Vous pourrez assurer le suivi de cette demande dans GESICO.",
+        "gauge": "gabarit",
+        "LongFRET": "Long parcours Fret (3 régions minimum, 4 en IDF)",
+        "maxSpeed": "vitesse max.",
+        "newGesicoRequest": "Nouvelle demande GESICO",
+        "pathType": "Type de parcours",
+        "pdf": "PDF",
+        "referenceEngine": "engin de référence",
+        "requestedRoute": "PARCOURS DEMANDÉ",
+        "retainedSimulation": "SIMULATION RETENUE",
+        "rollingStock": "matériel remorqué",
+        "selectPathType": "-- Choisir --",
+        "stop": "Arrêt de {{minutes}} à {{stopAt}}",
+        "successMessage": "Demande reçue par GESICO",
+        "toleranceAfter": "Tolérance future",
+        "toleranceBefore": "Tolérance passée",
```

See previous comment of @Caracol3 and @thibautsailly 

```suggestion
        "toleranceAfter": "Tolérance avale",
        "toleranceBefore": "Tolérance amont",
```

---

Done